README.md

```diff
@@ -199,6 +199,7 @@ search.addWidget(
     container: '#search-box',
     placeholder: 'Search for products',
     // cssClass
+    // poweredBy: boolean
```

Comment:
I would not make true the default mostly because we are targeting FIRST paying customers and that you do not want a powered by (by default) to show up when you just paid for something.

Then for our integrations/community experiences we can just use poweredBy: true. It will be even more explicit for us and communities.